kernel/byterun/coq_uint63_native.h

```diff
@@ -74,11 +74,12 @@
 
 /* addmuldiv(p,x,y) = x * 2^p + y / 2 ^ (63 - p) */
 /* (modulo 2^63) for p <= 63 */
-value uint63_addmuldiv(uint64_t p, uint64_t x, uint64_t y) {
+value uint63_addmuldiv(value p, value x, value y) {
+  if ((uint64_t)p >= 127) return uint63_zero();
```

Comment:
Could you explain the 127 here? According to the OCaml implem in kernel/uint63_63.ml I'd expected 64*2+1 = 129 here.
BTW, maybe permuting this with next line and having the test on `shiftby` would be more readable?

Reply:
Good catch. In fact, what you suggest is what I originally wrote, but then I decided that it would be more homogeneous to follow the same convention as the other operations earlier in the file.